src/index.html

```diff
@@ -45,11 +45,46 @@
       </div>
     </footer>
 
-    <script
-      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
-      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
-      crossorigin="anonymous"
-    ></script>
-    <script src="./index.js"></script>
+<!-- modal -->
+    <div id="item-modal" class="modal">
+      <div class="card-container">
+        <div class="header-container">
+            <div class="modal-header">
+              
+              <span class="close">&times;</span>
+            </div> 
+        </div>
+        <div class="modal-body">
+          <div class="image-container">
+            <img src="https://catholicreview.org/wp-content/uploads/2021/07/20210623T1100-MOVIE-REVIEW-F9-FAST-SAGA-1250672.jpg" alt="modal image">
```

Comment:
We should add the following details for the TV shows:

1. Title
2. Genre
3. Summary

[Optional]: we can add the Language also.